READ_BEFORE_UPDATE.md

```diff
@@ -51,26 +56,33 @@
 ### Migrating from privacyIDEA 3.9.2 to eduMFA 1.0.0
```

Comment:
Maybe it makes sense to shift this block to the top of the release notes and make it more generic? Migrating from privacyIDEA 3.9.2 to eduMFA